examples/pytorch/contrastive-image-text/run_clip.py

```diff
@@ -500,6 +537,12 @@ def filter_corrupt_images(examples):
         eval_dataset=eval_dataset if training_args.do_eval else None,
         data_collator=collate_fn,
     )
+    trainer.add_callback(TBTrainerCallback)
+    # Mlflow initial
+    #set the os enviroment for MLflowCallback
+    os.environ["DISABLE_MLFLOW_INTEGRATION"] = "False"
+    os.environ["HF_MLFLOW_LOG_ARTIFACTS"]="False"
```

Comment:
Do we have no problem getting all the MLFlow logs in the end even when it's false?

Reply:
The default option in the trainer is False, so there are no problems if we set them to False here. I just put them there to remind that we have the options.